.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: Rust
+
+on: [push, pull_request]
+
+jobs:
+  formatting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: Osspial/setup-rust-action@windows-uninstall-default-stable
+      with:
+        rust-version: stable
+        components: rustfmt
+    - name: Check Formatting
+      run: cargo +stable fmt --all -- --check
+
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        rust_version: [stable]
+        platform:
+          - { os: windows-latest, target: x86_64-pc-windows-msvc,   features: ',' }
+          - { os: windows-latest, target: i686-pc-windows-msvc,     features: ',' }
+          - { os: windows-latest, target: x86_64-pc-windows-gnu,    features: ',', host: -x86_64-pc-windows-gnu }
+          - { os: windows-latest, target: i686-pc-windows-gnu,      features: ',', host: -i686-pc-windows-gnu }
+          - { os: ubuntu-latest,  target: i686-unknown-linux-gnu,   features: ',' }
+          - { os: ubuntu-latest,  target: i686-unknown-linux-gnu,   features: ',' }
```

Comment:
Well, you are duplicating them in the matrix. This one is the same as the one above.

Reply:
🤦‍♀️